Tasks/JenkinsQueueJobV2/task.json

```
@@ -101,6 +101,15 @@
                 "resizable": "true",
                 "rows": "4"
             }
+        },
+        {
+            "name": "strictDefinition",
+            "type": "boolean",
+            "label": "Strict success definition",
+            "defaultValue": false,
+            "required": false,
+            "helpMarkDown": "Specifies strictness of a success definition: whether to consider unstable as a failure or not. False for non-strict, and true for strict version."
+
         }
```

Maybe we should move this input to the "advanced" group?

```suggestion
        {
            "name": "strictDefinition",
            "type": "boolean",
            "label": "Strict success definition",
            "defaultValue": false,
            "required": false,
            "helpMarkDown": "Specifies strictness of a success definition: whether to consider unstable as a failure or not. False for non-strict, and true for strict version.",
            "groupName": "advanced"
        }
```

---

Yeah, seems like a logical thing to do